pyquil/paulis.py

```diff
@@ -23,6 +23,7 @@
 import copy
 
 from typing import (
+    Any,
```

Comment:
I think this import can be dropped if coupled with other suggested changes elsewhere.

Reply:
To be fixed in the 2nd commit.